.github/workflows/stale.yaml

```diff
@@ -14,12 +14,16 @@ jobs:
     - uses: actions/stale@v4
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-pr-stale: 14
+        days-before-pr-close: 30
+        stale-pr-label: 'lifecycle/stale'
         stale-pr-message: >
           Marking this PR stale since there has been no activity for 14
-          days. It will be closed if there is no activity for another
-          30 days.
-        stale-pr-label: 'lifecycle/stale'
-        days-before-stale: 14
-        days-before-close: 30
-        days-before-issue-stale: -1 # disable marking issues as stale
-        days-before-issue-close: -1 # disable closing stale issues
+          days. It will be closed if there is no activity for 30 days.
```

Comment:
I actually think the previous message was correct, see e.g. https://github.com/projectcontour/contour/pull/4603 which was marked stale 26 days ago but hasn't been closed yet. Seems like the "close" counter starts when the issue/PR is labeled stale.

Reply:
oh oops i totally screwed up the math there whoops, will put back and fix the increments